config.html

```diff
@@ -228,7 +231,7 @@
                 <tr colspan="2">
                     <td>
                         <input type="checkbox" id="script_enabled" onclick="enableScript();"
-                            data-originalValue=""></input>
```

Comment:
Why did you remove closing of input tag?

Reply:
Hi SubZeroPL,

I removed the closing tags because they aren't actually necessary. See: [https://www.w3schools.com/tags/tag_input.asp](https://www.w3schools.com/tags/tag_input.asp)

Reply:
Oh, I didn't know that, thanks for clarification. Then it looks fine.